.github/workflows/main.yml

```diff
@@ -28,14 +28,15 @@ jobs:
       fail-fast: false
       matrix:
         typescript-version:
-          - 'latest'
-          - '~4.8.0'
-          - '~4.7.0'
+          - "latest"
+          - "~4.9.0"
```

Comment:
`type-fest` supports TypeScript 4.7+, so we should make sure to add coverage for TypeScript 4.9 too.